privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardRendererFactory.kt

```diff
@@ -53,6 +54,7 @@ class BrowserPrivacyDashboardRendererFactory @Inject constructor(
                 renderer.onBrokenSiteClicked,
                 renderer.onPrivacyProtectionsClicked,
                 renderer.onUrlClicked,
+                renderer.onOpenSettings,
```

Comment:
NIT: Same as with the viewmodel `Command`, could we rename to `onOpenAutoconsentSettings`?